tests/testthat/test-BFBayesFactor.R

```diff
@@ -19,67 +19,72 @@ test_that("p_direction: BF t.test two samples", {
   data(chickwts)
   chickwts <- chickwts[chickwts$feed %in% c("horsebean", "linseed"), ]
   chickwts$feed <- factor(chickwts$feed)
-  x <- ttestBF(formula = weight ~ feed, data = chickwts)
+  x <- BayesFactor::ttestBF(formula = weight ~ feed, data = chickwts)
   expect_equal(as.numeric(p_direction(x)), 1, tolerance = 1)
 })
 
 test_that("p_direction: BF t.test meta-analytic", {
   skip_if_not_or_load_if_installed("BayesFactor")
-  t <- c(-.15, 2.39, 2.42, 2.43)
+  t <- c(-0.15, 2.39, 2.42, 2.43)
   N <- c(100, 150, 97, 99)
-  x <- meta.ttestBF(t = t, n1 = N, rscale = 1)
+  x <- BayesFactor::meta.ttestBF(t = t, n1 = N, rscale = 1)
   expect_equal(as.numeric(p_direction(x)), 0.99975, tolerance = 1)
 })
 
+skip_if_not_or_load_if_installed("BayesFactor")
+
 # ---------------------------
 # "BF ANOVA"
 data(ToothGrowth)
 ToothGrowth$dose <- factor(ToothGrowth$dose)
 levels(ToothGrowth$dose) <- c("Low", "Medium", "High")
 x <- BayesFactor::anovaBF(len ~ supp * dose, data = ToothGrowth)
 test_that("p_direction", {
-  expect_equal(as.numeric(p_direction(x)), 91.9, tol = 0.1)
+  expect_equal(as.numeric(p_direction(x)), c(1, 0.95675, 0.95675, 1, 1), tolerance = 0.1)
 })
 
 # BF ANOVA Random ---------------------------
 
 data(puzzles)
 x <- BayesFactor::anovaBF(RT ~ shape * color + ID, data = puzzles, whichRandom = "ID")
 test_that("p_direction", {
-  expect_equal(as.numeric(p_direction(x)), 91.9, tol = 0.1)
```

Comment:
@DominiqueMakowski @mattansb @IndrajeetPatil The results of these tests seem to have changed substantially. I hope it is still correct? Not sure whether these were undetected changed in _BayesFactor_ pkg or is there an error we made when addressing #618 / #619 or something?

Reply:
I checked and the output of as.numeric is 

```
[1] 1.00000 0.98375 0.98375 0.99375 0.68750 0.83100 0.68325 0.99825 0.66925 0.99900 0.60375 0.99600 0.75925 0.76725 0.99925 1.00000
[17] 1.00000 1.00000
```

I think the test is wrong

Reply:
Better to replace with e.g.,:
```r
out <- p_direction(x)
expect_equal(out[out$Parameter == "shape-round", "pd"], 0.98375, tol = 0.1)
```

Reply:
> I think the test is wrong

Yes, the former test just returned a single value (i.e. one parameter), now we have a bunch of parameters (and values). I changed the test, but what's the reason that the result of the test changed that much? ❓

Reply:
Very weird indeed. Must be a very old test - it also has the probability retuned as a percent value and not a proportion.